### ISSUE TITLE
Debian package build depends on outdated libssl

### ISSUE THREAD

**Issue by baconwaifu** [alberthdev/spasm-ng#56]
libssl1.0.0 is unavailable in modern debian versions, but it builds just fine on the more recent 1.1.

**Comment by alberthdev** [comment]
Agreed. I'm assuming you're referring to the Debian packages, right?

These are built with the checkinstall tool called from [here](https://github.com/alberthdev/spasm-ng/blob/master/Makefile#L76), so if we just conditionally change the package name to libssl1.1 it should work. Could you test that out and see if you can generate a working package for newer versions of Debian? (PR appreciated if you have the time!)

Long term we should probably have something like [OBS](https://openbuildservice.org/) handle the builds instead of us doing it ourselves...
